test/smoke/src/areas/positron/apps/shiny.test.ts

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import { join } from 'path';
+import { Application, Logger, PositronPythonFixtures, PositronRFixtures } from '../../../../../automation';
+import { installAllHandlers } from '../../../utils';
+import { expect } from '@playwright/test';
+
+export function setup(logger: Logger) {
+	describe('Shiny Application', () => {
+		// Shared before/after handling
+		installAllHandlers(logger);
+
+		before(async function () {
+			await this.app.workbench.extensions.installExtension('posit.shiny', true);
+
+			await this.app.workbench.extensions.closeExtension('Shiny');
+		});
+
+		describe('Shiny Application - Python', () => {
+			before(async function () {
+				await PositronPythonFixtures.SetupFixtures(this.app as Application);
+			});
+
+			it('Python - Verify Basic Shiny App [...]', async function () {
+				const app = this.app as Application;
+
+				await app.workbench.quickaccess.openFile(join(app.workspacePathOrFolder, 'workspaces', 'shiny-py-example', 'app.py'));
```

Comment:
Good question.  Its kind of buried in the framework:
https://github.com/posit-dev/positron/blob/main/test/smoke/src/main.ts#L161
https://github.com/posit-dev/positron/blob/main/test/smoke/src/main.ts#L276-L277

So that [repo](https://github.com/posit-dev/qa-example-content/tree/main) is made available to all tests by the "global startup"